rancher/install-rancher.sh

```diff
@@ -3,16 +3,17 @@
 
 set -x
 
-if [ "$#" -ne 4 ]
+if [ "$#" -ne 5 ]
 then
-	echo "Usage $0 <hostname> <email> <clproject> <apikey>"
+	echo "Usage $0 <hostname> <email> <clproject> <apikey> <bootstrappassword>"
 	exit 1
 fi
 
 HOST=$1
 EMAIL=$2
 PROJECT=$3
 APIKEY=$4
+BOOTSTRAPPASSWORD=$5
```

Comment:
om denna var ett problem i prod, hade vi kunnat slumpa den istället för att kunna få ut detta innan Cloud? 
fattar det som att den går att få ut i efterhand: https://rancher.com/docs/rancher/v2.6/en/installation/resources/bootstrap-password/

Reply:
verkar som att det slumpas om man inte sätter det?

Reply:
Skickat in en ny commit som aktiverar kubernetes configen i .bashrc på användaren som skapar servern.